src/sentry/utils/committers.py

```diff
@@ -39,8 +39,7 @@ def score_path_match_length(path_a, path_b):
     return score
 
 
-def _get_frame_paths(event):
-    data = event.data
```

Comment:
Do we use the `data` anywhere else? I think what @JTCunning was going for here is that we'd extract just the `stacktrace` and pass that along so that we minimize the data we send.

Reply:
Alright, I refactored it so I call `get_frame_paths` in post_process and just pass the result of that to the task.